app/services/document_publishing_service.rb

```diff
@@ -17,6 +17,7 @@ def publish(document, review_state)
     document.update!(publication_state: "sending_to_live", review_state: review_state)
     publishing_api.publish(document.content_id, nil, locale: document.locale)
     document.update!(publication_state: "sent_to_live", change_note: nil, update_type: "major", has_live_version_on_govuk: true)
+    publish_images(document.images)
```

Comment:
Should image publishing happen before the sending to the publishing-api?

Reply:
I think you're right, it probably makes sense for the images to be published before the document is.